packages/react-data-grid-addons/package-lock.json

```diff
@@ -2558,6 +2546,12 @@
 				"object-assign": "^4.1.0"
 			}
 		},
+		"react-data-grid": {
```

Comment:
Might want to run `npm install` from the root directory to get rid of this.

Reply:
npm install does not fix it